app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt

```diff
@@ -84,7 +87,17 @@ class SettingsFragment : Fragment(), SettingsView, AdapterView.OnItemClickListen
                 shareIntent.putExtra(Intent.EXTRA_TEXT, shareSub)
                 startActivity(Intent.createChooser(shareIntent, getString(R.string.msg_share_using)))
             }
+            resources.getString(R.string.title_rate_us) -> startAppPlayStore()
+        }
+    }
 
+    private fun startAppPlayStore() {
+        try {
+            fun getMarketAppLink()= getString(R.string.market_link)
```

Comment:
There is no need to put these functions inside `startAppPlayStore()`. What I mean is to put them under the `startAppPlayStore` not inside. 

Instead of creating these two function to return the market app link and market web link you can eliminate them and do that: `getString(R.string.market_link).toUri()` and `getString(R.string.play_store_link).toUri()` respectively. 
This is another option too.

Reply:
Ok, I will go with second option 😀